From 045e88bcc4795ec01793871d94570bb9f0a68d3f Mon Sep 17 00:00:00 2001
From: jpadmana <jayashree.r.padmanaban@intel.com>
Date: Mon, 17 Sep 2012 14:20:16 +0530
Subject: [PORT FROM MAIN] Issues found in offload Pre int build testing - Adding EARPIECE support for offload

BZ: 57562

Added EARPIECE offload support

Change-Id: I1777b3087cc4f9a1462e2b8ddd559ec00b2ba1dd
Orig-Change-Id: I6e30a86dd65135f9e28b6ae5b527dce811fe86e9
Signed-off-by: jpadmana <jayashree.r.padmanaban@intel.com>
Reviewed-on: http://android.intel.com:8080/78749
Reviewed-by: M, Arulselvan <arulselvan.m@intel.com>
Tested-by: M, Arulselvan <arulselvan.m@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 include/system/audio.h |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/include/system/audio.h b/include/system/audio.h
index e53b287..6d0f9eb 100644
--- a/include/system/audio.h
+++ b/include/system/audio.h
@@ -374,7 +374,8 @@ enum {
     //devices not supported for codec offload
     AUDIO_DEVICE_OUT_NON_OFFLOAD = (AUDIO_DEVICE_OUT_ALL & ~(AUDIO_DEVICE_OUT_SPEAKER |
                                  AUDIO_DEVICE_OUT_WIRED_HEADSET |
-                                 AUDIO_DEVICE_OUT_WIRED_HEADPHONE)),
+                                 AUDIO_DEVICE_OUT_WIRED_HEADPHONE |
+                                 AUDIO_DEVICE_OUT_EARPIECE)),
 };
 
 typedef uint32_t audio_devices_t;
-- 
1.7.4.1

